core/domain/rte_component_registry_test.py

```diff
@@ -55,15 +55,15 @@
 class RteComponentUnitTests(test_utils.GenericTestBase):
     """Tests that all the default RTE components are valid."""
 
-    def _is_camel_cased(self, name):
+    def is_camel_cased(self, name):
```

Comment:
is there any reason for changing this (and other similar) private method to the public? (We can't change a private method to the public just for testing purposes.)

/cc @seanlip, @ankita240796